modules/intentIqAnalyticsAdapter.md

```diff
@@ -0,0 +1,27 @@
+# Overview
+
+Module Name: iiqAnalytics
+Module Type: Analytics Adapter
+Maintainer: julian@intentiq.com
+
+# Description
+
+By using this Intent IQ adapter, you will be able to obtain comprehensive analytics and metrics regarding the performance of the Intent IQ Unified ID module. This includes how the module impacts your revenue, CPMs, and fill rates related to bidders and domains.
+
+## Intent IQ Universal ID Registration
+
+No registration for this module is required.
+
+## Intent IQ Universal IDConfiguration
+
+<B>IMPORTANT</B>: requires Intent IQ Universal ID module be installed and configured. [(How-To)](https://docs.prebid.org/dev-docs/modules/userid-submodules/intentiq.html)
```

Comment:
instead of saying requires, perhaps say 'only effective when' or something of this nature. Modules aren't supposed to require each other.